app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/UpdateVatSubscription.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatsubscription.models.updateVatSubscription
+
+import play.api.libs.json._
+
+case class UpdateVatSubscription(requestedChanges: RequestedChanges,
+                                 updatedReturnPeriod: UpdatedReturnPeriod,
```

Comment:
Seems like updatedReturnPeriod should be `Optional[UpdatedReturnPeriod]` as going forward there may, or may not, be a change for it (depending non the boolean value in `requestedChanges`)

I know we're adding this in for the `ChangeReturnPeriod` story but can we future proof the design of the `UpdateVatSubscription` model